CONTRIBUTING.md

```diff
@@ -0,0 +1,183 @@
+# Contributing
+
+- [Issues and PRs](#issues-and-prs)
+- [Submitting a pull request](#submitting-a-pull-request)
+    - [Helm version](#helm-version)
+    - [Best Practices](#best-practices)
+    - [Commit Guidelines](#commit-guidelines)
+    - [Tests](#tests)
+        - [Unit Tests](#unit-tests)
+            - [Golden Files](#golden-files)
+            - [Properties Test](#properties-test)
+        - [Test License Headers](#test-license-headers)
+        - [OpenShift](#openshift)
+    - [Documentation](#documentation)
+- [CI](#ci)
+    - [OpenShift](#openshift-1)
+- [Resources](#resources)
+
+[fork]: /fork
+[pr]: /compare
+[CODE_OF_CONDUCT]: CODE_OF_CONDUCT.md
+
+We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great.
+
+Please note that this project is released with a [Contributor Code of Conduct](https://github.com/camunda/camunda-platform-helm/blob/main/CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
+
+## Issues and PRs
+
+If you have suggestions for how this project could be improved, or want to report a bug, open an issue! We'd love all and any contributions. If you have questions, too, we'd love to hear them.
+
+We'd also love PRs. If you're thinking of a large PR, we advise opening up an issue first to talk about it, though! Look at the links below if you're not sure how to open a PR.
+
+## Submitting a pull request
+
+Please feel free to fork this repository and open a pull request to fix an issue or add a new feature.
+
+Make sure that your provided PR's works via:
+
+- `helm lint` to run the linting
+- `make go.fmt` to run the gofmt
+- `make go.addlicense-check` to run the license check
+- `make go.test` to run the go tests
+- `helm install <releasename> chartPath/` to install a Helm release in your K8s cluster (e.g. kind)
+
+We have the following expectation on PR's:
+
+- They follow the [best practices](#best-practices)
+- They contain new [tests](#tests) on a bug fix or on adding a new feature
+- They follow the [commit guidelines](#commit-guidelines)
+- The [documentation](#documentation) has been updated, if necessary.
+
+### Helm version
+
+To have a smooth contribution experience, before working on a new PR make sure to use the exact Helm version
+that's currently used in the repo.
+
+Helm version is set in the [.tool-versions](./.tool-versions) file, so you can use [asdf version manager](https://github.com/asdf-vm/asdf) to install Helm locally or just install the same version manually.
```

Comment:
:thought_balloon: Just noticed that we should probably use the same version for CI. There we currently specify 3.11 while `.tool-versions` requires 3.9.